test/testcases.py

```diff
@@ -41,19 +41,22 @@ def __str__(self):
 class TestCaseFedora(TestCase):
     container_ref: str = "quay.io/fedora/fedora-bootc:40"
     rootfs: str = "btrfs"
+    osinfo_template: str = "Fedora Server 40 (%s)"
```

Comment:
(nitpick) Might be worthwhile to add a small comment that the %s will be replaced with the architecture

Reply:
I used a different string formatting so the comment is not needed, please take a look

Reply:
Love it, thank you!